contracts/state_rent/IStateRent.sol

```diff
@@ -23,4 +29,19 @@ interface IStateRent {
     // Transfer the maintainer of the contract.
     // Returns a boolean value indicating whether the operation succeeded.
     function transferMaintainer(address contract_address, address new_maintainer) external returns (bool);
+
+    // Deploy contract
+    // Returns a boolean value indicating whether the operation succeeded.
+    function deployContract(address contract_address) external returns (bool);
```

Comment:
do we want to rename it to publish contract instead? or a better name? because otherwise this could be confused with the create contract